pages/blog/the-new-era-approaches.md

```diff
@@ -0,0 +1,256 @@
+    turnOnPayload:
+      type: object
+      properties:
+        command:
+          type: string
+          enum:
+            - on
+          description: Whether to turn on the light.
+        streetlightId:
+          description: The ID of the streetlight.
+          type: string
+        sentAt:
+          $ref: "#/components/schemas/sentAt"
+
+    turnOnReplyPayload:
+      type: object
+      properties:
+        turnedOnTimestamp:
+          type: string
+          format: date-time
+          description: Date and time when the light was actually turned on.
+
+    lightMeasuredPayload:
+      schemaFormat: "application/vnd.apache.avro;version=1.9.0"
+      schema:
+        type: record
+        name: User
+        namespace: com.company
+        doc: User information
+        fields:
+          - name: lumens
+            type: int
+          - name: sentAt
+            type: timestamp_ms
+
+  securitySchemes:
+    saslScram:
+      type: scramSha256
+      description: Provide your username and password for SASL/SCRAM authentication
+
+  parameters:
+    streetlightId:
+      description: The ID of the streetlight.
+
+  messageTraits:
+    commonHeaders:
+      headers:
+        type: object
+        properties:
+          my-app-header:
+            type: integer
+            minimum: 0
+            maximum: 100
+
+  operationTraits:
+    kafka:
+      bindings:
+        kafka:
+          clientId: public
+```
+As of the pre-release `v3.0.0-next-major-spec.12`, this is a valid AsyncAPI document. You can always find the most recent pre-release version here: https://www.asyncapi.com/docs/reference
+
+All the changes in 3.0 up until now are the following:
+
+- Request/reply pattern
+- Introduce the new Channel Object, detached from operations
+- Introduce the new Operation object, detached from channels
+- Channels are no longer identified with address/topic/path.
+- Optional channels
+- Schemas and schema formats are now naturally bound
+- Cleaned up the root object
+- Added additional meta fields for Server Object, Channel Object, Operation Object, and Operation Trait Object,
+- External Documentation Object and Tag Object can now be reused and referenced.
+- Unified referencing behavior
+
+In due time we will give you a complete rundown about all the changes in 3.0, as well as extended documentation that explains the features in more in-depth, including a migration guide and release blog post.
+
+## The Remaining Effort
+
+The specification work is nearly done, only one change is still being discussed which is changing traits behavior to an inheritance that can be overwritten.
+
+However, at AsyncAPI, a specification is nothing without documentation and tools, which is why the majority of the remaining effort resolves just that.
+
+For documentation, you have probably noticed that since the first release of 2.0, we now have concepts, tutorials, and guides, and almost all is going to change for 3.0 in some way or another.  
```

Comment:
```suggestion
For documentation, you have probably noticed that since the first release of 2.0, we now have concepts, tutorials, and guides. Some of those docs will be updated due to 3.0.  
```

Actually, there is not that many changes to make, so fixing this line. I have confirmed it myself and also Sergio.